core-war/src/main/java/org/silverpeas/web/socialnetwork/myprofil/control/MyProfilSessionController.java

```diff
@@ -229,4 +248,22 @@ public boolean updatablePropertyExists() {
   public void unlinkSocialNetworkFromSilverpeas(SocialNetworkID networkId) {
     SocialNetworkService.getInstance().removeExternalAccount(getUserId(), networkId);
   }
-}
\ No newline at end of file
+
+  private void processDataOfExtraTemplate(HttpRequest request) {
+    PublicationTemplateManager templateManager = PublicationTemplateManager.getInstance();
+    PublicationTemplate template = templateManager.getDirectoryTemplate();
+    if (template != null) {
+      try {
+        PagesContext context = getTemplateContext();
+        templateManager.saveData(template.getFileName(), context, request.getFileItems());
+      } catch (Exception e) {
+        MessageNotifier.addError("Les données du formulaire n'ont pas été enregistrées !")
+            .setDisplayLiveTime(10000);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 10000 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)